app/ui/src/app/support/support.component.scss

```diff
@@ -1,19 +1,11 @@
-.mainform {
-    background-color: white;
-    padding: 2em;
-}
+@import 'patternfly-sass/assets/stylesheets/patternfly/variables';
```

Comment:
If we feel we'll want to refer to this SASS partial often, then let's add it to `scss/base/_vendor.scss` and then import `syndesis-sass` instead. This way we will not have to import the same partial again and again in the future